components/fxa-client/src/commands/send_tab.rs

```diff
@@ -40,16 +42,31 @@ impl EncryptedSendTabPayload {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct SendTabPayload {
     pub entries: Vec<TabHistoryEntry>,
+    // XXX - tests don't fail yet even though none add these 2 new items and
+    // they aren't marked as optional for serde, which seems odd!
```

Comment:
Do you intend to mark them optional before landing though? (I agree the lack of failures seems odd in the meantime)

Reply:
I do - this is more a TODO for myself to work out if test coverage really is missing, and if so, add it.